Shoko.Server/Extensions/ModelClients.cs

```diff
@@ -22,8 +23,8 @@ public static CL_ServerSettings ToContract(this IServerSettings settings)
         {
             AniDB_Username = settings.AniDb.Username,
             AniDB_Password = settings.AniDb.Password,
-            AniDB_ServerAddress = settings.AniDb.ServerAddress,
-            AniDB_ServerPort = settings.AniDb.ServerPort.ToString(),
+            AniDB_ServerAddress = Regex.Match(settings.AniDb.HTTPServerUrl, @"(https?://)*[^:]+").ToString(),
```

Comment:
Should the regex be `@"(https?://).*?[^:]+")` instead? Unless c# regex works differently to what I'm used to, it's the `https://` capture group that it tries to match 0+ times otherwise.

whacked a few test urls into https://regex101.com/ using the above and I _think_ my suggestion isn't baseless

Reply:
Testing the existing regex on single line strings... Does work as expected :)

I'm the fool that tried being lazy and testing on multiline strings without adapting the regex accordingly.

Reply:
Either is right if you didn't know what I was expecting it to do. Such is the way of regex 😔